### ISSUE TITLE
Breaking change in latest @grpc/grpc-js version

### ISSUE THREAD

**Issue by pierback** [deeplay-io/nice-grpc#32]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `nice-grpc@1.0.4` for the project I'm working on.

With the latest version update of `@grpc/grpc-js`, the enum `ConnectivityState` was moved from `@grpc/grpc-js/build/src/channel` to `@grpc/grpc-js/build/src/connectivity-state` and thus throwing the error `Failed to connect to ETF service: dns:localhost:5001 TypeError: Cannot read property 'READY' of undefined` when a grpc client tries to establish a connection with the corresponding server. 

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/nice-grpc/lib/client/channel.js b/node_modules/nice-grpc/lib/client/channel.js
index 27a55dd..6abfc71 100644
--- a/node_modules/nice-grpc/lib/client/channel.js
+++ b/node_modules/nice-grpc/lib/client/channel.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.waitForChannelReady = exports.createChannel = void 0;
 const grpc_js_1 = require("@grpc/grpc-js");
-const channel_1 = require("@grpc/grpc-js/build/src/channel");
+const channel_1 = require("@grpc/grpc-js/build/src/connectivity-state");
 function createChannel(address, credentials, options = {}) {
     const match = /^(?:([^:]+):\/\/)?(.*?)(?::(\d+))?$/.exec(address);
     if (match == null) {
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by aikoven** [comment]
Thanks for the report.

I noticed that they actually re-export `ConnectivityState` under a bit different name: https://github.com/grpc/grpc-node/blob/81ef5e33c4bb97d97cf9112ffe60663d14bdba87/packages/grpc-js/src/index.ts#L129

Will change the import to use it.

**Comment by aikoven** [comment]
Fixed in 📦  `nice-grpc@1.0.5`

**Comment by pierback** [comment]
Thanks a lot 🙏🏻
